sources/index.yml

```diff
@@ -1440,7 +1451,7 @@ jobs:
             echo "${allWorkspaces[@]}"
             echo ""
 
-            workspacesToDestroy="$(echo "${allWorkspaces[@]}" | commitWorkspaces | withoutOutdatedCommits)"
+            workspacesToDestroy="$(echo "${allWorkspaces[@]}" | commitWorkspaces | withoutOutdatedCommits | filterOutdatedWorkspacesByAge)"
```

Comment:
If this is a filter that's just added to the existing filters. then won't this only limit when we destroy things, not add a new condition?

Like rather than:

"Delete workspaces like we did before, but also if they're older than 14 days"

This will be like:

"Delete workspaces only when the previous conditions are met AND they're older than 14 days"

no?

Reply:
@TerranceN Yes, good catch. Idk what I was thinking. I think I've fixed this now though.